client/src/components/Home/HomeReducer.js

```diff
@@ -0,0 +1,8 @@
+export default (users = [], action) => {
+    switch(action.type) {
+        case 'FETCH_ALL':
```

Comment:
Please add a new file say constants.js and export 'FETCH_ALL' as a constant and use it here and where ever is required.
`export const FETCH_ALL = 'FETCH_ALL'`;
And please be descriptive enough for readability. eg: 'FETCH_ALL_<WHAT>'